docs/user-guide/admin-cli.md

```diff
@@ -11,13 +11,61 @@ of all virtual disks created and in use on the host.  For these reasons, an admi
 created that runs on the ESXi host and that provides access to information not visible from the
 Docker CLI.
 
-The admin cli also enables ESX admins to implement tenancy.
+The admin CLI also enables ESX admins to implement access control and basic storage quotas.
+
+Admin CLI is located at `/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py`. It supports `--help`  at
+every command and sub-command. e.g.
+
+**NOTE** PXE Boot of ESX host is not supported, as the code relies on configuration / authorization databse (aka "config DB") (or symlink) being in /etc/vmware/vmdksops/auth-db. If the host is PXE  booted, access control will be turned off, as the code will interpret the lack of DB or link at the above address as an instruction to provide no access control.
```

Comment:
Can't the admin CLI or the service enumerate and search databases on startup to figure if any are hosting a config DB and use that? Stateless ESX may be used by multiple users who should be able to use this feature.